example/mysql/simplecrud-example.pl

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/perl
+
+use lib '../lib';
+use Dancer;
+use Dancer::Plugin::SimpleCRUD;
+
+simple_crud(
+    record_title => 'Person',
+    db_table => 'people',
+    #db_connection_name => 'foo',
+    prefix => '/people',
+    acceptable_values => {
+        gender => [ qw( Male Female ) ],
+		is_admin => [ qw( 0 1 ) ],
+    },
+    deletable => 'yes',
+    sortable => 'yes',
+    paginate => 3,
+    downloadable => 1,
+    foreign_keys => {
+        employer_id => {
+            table        => 'employer',
+            key_column   => 'id',
+            label_column => 'name',
+        },
+    },
+    table_class => 'table table-responsive',
+    #custom_columns => {
+     #   mailto_link => {
+      #      raw_column => 'email',
+       #     transform  => sub { my $email = shift; return "<a href='mailto:$email'>mail</a>"; },
+        #},
+        #full_name => {
+         #   raw_column => "(first_name || ' ' || last_name)",
+          #  transform => sub { return shift }, # (unnecessary, btw, as this is the default)
+        #},
+    #},
+    labels => {
+        age => 'Age (years)',
+    },
+    auth => {
+        view => {
+            require_login => 1,
+        },
+        edit => {
+            require_role => 'editor',
+        },
+    },
+);
+
+get '/' => sub {
+    redirect '/people';
+};
+
+# redirect for add new register in db_table arg
+get '/add' => sub {
+    redirect '/people/add';
+};
```

Comment:
I think these redirects make it difficult for people to use the provided template and add more simplecrud interfaces.